x-pack/plugins/canvas/canvas_plugin_src/functions/common/plot/get_tick_hash.ts

```diff
@@ -19,7 +19,7 @@ export const getTickHash = (columns: DatatableColumn[], rows: DatatableRow[]) =>
     },
   };
 
-  if (get<DatatableColumn[], DatatableColumnType>(columns, 'x.type') === 'string') {
+  if (get(columns, 'x.type') === 'string') {
```

Comment:
Is there any any advantage to using `get` in scenarios like this, or, since `columns` is properly typed, should we just do `if (columns.x.type === 'string') ...`

Reply:
Good point, but I'm trying to avoid changing logic too much, particularly since  callsites are likely not TS'd, so the strong typing doesn't prevent bad calls.